client/state/sharing/publicize/schema.js

```diff
@@ -0,0 +1,55 @@
+export const fetchingConnectionsSchema = {
```

Comment:
Another consistency point, we should have a common approach to exporting schemas, since this differs from how I had named and used the exports in #3251. Differences boil down to:

```js
// #3251
export const fetchingConnections = {};
import * as schema from './schema';
isValidStateWithSchema( schema.fetchingConnections );

// #3357
export const fetchingConnectionsSchema = {};
import { fetchingConnectionsSchema } from './schema';
isValidStateWithSchema( fetchingConnectionsSchema );
```

I'd probably lean more towards your approach to enable importing specific values without concern for naming conflicts (`import as` is an alternative, albeit a bit ugly).

Another point to make is that I think we should never `export default` from schemas unless the reducer itself is not a combined reducer but rather a simple function (which is not what we're doing with [sites schema](https://github.com/Automattic/wp-calypso/blob/master/client/state/sites/schema.js)).